### PR TITLE
Switch to Mousetrap shortcut library

### PR DESCRIPTION
**Pull request by charlielee**
This stops the bugs that were occurring with the previous shortcuts being global.

In addition:
* The shortcuts displayed in the top menu bar are now taken from the first shortcut listed for each feature in `default-shortcuts.json`.
* Menubar functionality has been refactored and moved to `menubar.js`

### CODE REVIEW

**Comment by le717** [comment]
When are you wanting to merge this?

File: install.js [hunk 1]
```diff
@@ -10,8 +10,16 @@ fs.exists("app/lib", function(exists) {
...
   fs.copy("node_modules/mkdirp/index.js", "app/lib/mkdirp.js", { replace: true }, function (err) {
     console.log(err ? err : "Copied mkdirp to 'app/lib/mkdirp.js'");
   });
+  // Mousetrap
+  fs.copy("node_modules/mousetrap/mousetrap.min.js", "app/lib/mousetrap.js", { replace: true }, function (err) {
+    console.log(err ? err : "Copied mousetrap.min.js to 'app/lib/mousetrap.js'");
+  });
+  fs.copy("node_modules/mousetrap/plugins/pause/mousetrap-pause.min.js", "app/lib/mousetrap-pause.js", { replace: true }, function (err) {
+    console.log(err ? err : "Copied mousetrap-pause.min.js to 'app/lib/mousetrap-pause.js'");
+  });
```
Comment (le717) [inline install.js]: Just going to add that we have `app/js/file.js` available for use you might consider instead of the raw `fs` methods :)

**Comment by charlielee** [comment]
This is ready to go! There are some things related to shortcuts I'd like to refine, such as making every shortcut's `click` function refer to specific methods rather than being anonymous functions but I see this as outside of the scope of this PR.

**Comment by le717** [comment]
> such as making every shortcut's click function refer to specific methods rather than being anonymous functions but I see this as outside of the scope of this PR

Yea, that's what "Refactor JavaScript into modules" on the roadmap means. It's a gradual, continuous process, one this PR helps push towards. :)

If you've already tested it all, I'll go ahead and merge it. I can't test it now but I'm sure you have.

**Comment by le717** [comment]
I just pulled this:
```
Failed to load resource: net::ERR_FILE_NOT_FOUND
chrome-extension://igfmookapjjkefbjiccfdlcnjcmdoabo/app/lib/mousetrap-pause.js Failed to load resource: net::ERR_FILE_NOT_FOUND
Uncaught ReferenceError: Mousetrap is not defined
    at C:\Users\Caleb\Documents\Code\boats-animator\app\js\shortcuts.js:81:11
    at Array.forEach (native)
    at C:\Users\Caleb\Documents\Code\boats-animator\app\js\shortcuts.js:76:54
    at Array.forEach (native)
    at Object.addShortcuts [as add] (C:\Users\Caleb\Documents\Code\boats-animator\app\js\shortcuts.js:74:44)
    at chrome-extension://igfmookapjjkefbjiccfdlcnjcmdoabo/app/js/main.js:172:15
    at Object.success (C:\Users\Caleb\Documents\Code\boats-animator\app\js\shortcuts.js:145:9)
    at C:\Users\Caleb\Documents\Code\boats-animator\app\js\file.js:101:18
    at tryToString (fs.js:425:3)
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:412:12)
```

**Pull request merged**